Dockerfile

```
@@ -12,12 +12,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ffmpeg \
     && rm -rf /var/lib/apt/lists/*
 
-RUN pip install markitdown
+RUN pip install markitdown fastapi uvicorn
```

```suggestion
RUN pip install markitdown fastapi uvicorn python-multipart
```

Without `python-multipart` the API won't start.

It's still weird that `markitdown` is being installed here. This means if the source gets changed and you want to check if the Docker image works you first need to publish the package to pip... It would be better if the current `src` folder would be included here.

---

When you install with pip install "fastapi[standard]" it comes with some default optional standard dependencies.

If you don't want to have those optional dependencies, you can instead install pip install fastapi.